web/job_specs_controller.go

```diff
@@ -28,20 +27,16 @@ func (jsc *JobSpecsController) Index(c *gin.Context) {
 		return
 	}
 
-	var order func(opts *index.Options)
+	var order orm.SortType
```

Comment:
Seems very similar to the JobRunsController [functionality](https://github.com/smartcontractkit/chainlink/pull/863/files#diff-3a83a923e9b38bdd62896aebf215c1b5R35). Seems like we could pull this out into a helper method for the web package.

Reply:
I'll try to do that when we switch to `gorm`.